packages/http/src/mocker/HttpMocker.ts

```diff
@@ -37,46 +37,71 @@ export class HttpMocker implements IMocker<IHttpOperation, IHttpRequest, IHttpCo
       throw new Error('Http request is not defined');
     }
 
-    // setting default values
-    const acceptMediaType = input.data.headers && caseless(input.data.headers).get('accept');
-    config = config || { mock: false };
-    const mockConfig: IHttpOperationConfig =
-      config.mock === false ? { dynamic: false } : Object.assign({}, config.mock);
+    let payloadGenerator: PayloadGenerator = generateStatic;
 
-    if (!mockConfig.mediaTypes && acceptMediaType) {
-      mockConfig.mediaTypes = acceptMediaType.split(',');
+    if (config && typeof config.mock !== 'boolean' && config.mock.dynamic) {
+      payloadGenerator = generate;
```

Comment:
can't we just do:
```
const payloadGenerator = (config && typeof config.mock !== 'boolean' && config.mock.dynamic) && generate;
```
?

Reply:
Not exactly, but I've been able to refactor it in a way very similar. ffffb9d